packages/scandipwa/src/util/Validator/Validator.js

```diff
@@ -86,6 +88,16 @@ export const validate = (value, rule) => {
             }
         }
     }
+
+    if (fileExtension && value !== '') {
+        const { accept } = fileExtension;
+        const acceptedExtensions = accept.split(', ');
+        const currentFileExtension = value.split('.').pop();
+
+        if (acceptedExtensions.filter((extension) => extension === currentFileExtension).length === 0) {
```

Comment:
This could be replaced with `acceptedExtensions.includes(currentFileExtension)`
A lot easier to read and shorter

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes